src/clj/game/core/ice.clj

```diff
@@ -534,6 +534,16 @@
               (map :label (sort-by :index broken-subs)))
         "\")")))
 
+(defn break-subs-event-context
+  [state _ ice broken-subs breaker]
+  {;; pass a few more pieces of information relevant to other cards
+   :outermost (when-let [server-ice (:ices (card->server state ice))] (same-card? ice (last server-ice)))
+   :during-run (if (:run @state) true nil)
+   :all-subs-broken (all-subs-broken? ice)
+   :broken-subs broken-subs
+   :breaker (:cid breaker)
+   :card ice})
```

Comment:
This could be `:ice` to make it really clear what the context is. `:card` is fairly generic so that's what I've defaulted to in the past, but I think checking `(:ice context)` and `(:breaker context)` are more clear.